### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by vistajess**

### CODE REVIEW

**Comment by vistajess** [comment]
layout

**Comment by vistajess** [comment]
layout

File: styles/components/color.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.-info {
+  background-color: $brand-sky;
```
Comment (srph) [inline styles/components/color.scss:2]: huh? why do we have this?

File: styles/components/color.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+}
+.-warning {
```
Comment (srph) [inline styles/components/color.scss:4]: Like I said, we have `.-sun`, not `.-warning`.

File: docs/styles/layout.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.doc-block-grid {
+  width : 100%;
```
Comment (srph) [inline docs/styles/layout.scss:2]: - Add a bg here :confused: 
- Also, keep in mind to have no space between the property declaration and the colon.

``` sass
// bad
width : 100%

// good
width: 100%
```

**Comment by srph** [comment]
Thanks! 

**Note** Alhough this has issues, I'll merge this so I can start working on it.

File: docs/sections/Docs-Layout/index.jsx [hunk 4]
```diff
@@ -5,6 +5,142 @@ export default class Layout extends React.Component {
     return (
       <div>
         <h1 className="doc-heading">Layout</h1>
+        <div className="row doc-bottom-space _clearfix">
```
Comment (srph) [inline docs/sections/Docs-Layout/index.jsx:8]: No need to use `._clearfix` now with `.row`. I just fixed `.row`.

**Comment by vistajess** [comment]
Thanks I'll just fix then.. :)

**Comment by srph** [comment]
It's already fixed :smile:

**Pull request merged**